.github/workflows/ci.yml

```diff
@@ -90,6 +106,16 @@ jobs:
           git clone https://github.com/avantifellows/plio-analytics
           cd plio-analytics/
 
+          # check branch and switch to branch if exists
+          if [ `git branch --list --remote origin/${{ github.head_ref }}` ]
+          then
+            echo "Switching to branch ${{ github.head_ref }}."
+            git checkout ${{ github.head_ref }}
+            git pull origin ${{ github.head_ref }}
+          else
+            echo "Branch not found. Going with default branch."
+          fi
+
```

Comment:
same comment on reusing the output of this step as mentioned in [avantifellows/plio-analytics#26](https://github.com/avantifellows/plio-analytics/pull/26)